CHANGES.md

```diff
@@ -1,3 +1,43 @@
+## edge-19-2.4
+
+* CLI
+  * Implemented `proxy-cpu-limit` and `proxy-memory-limit` for setting the
+    proxy resources limits (`proxy-cpu` and `proxy-memory` were deprecated in
+    favor of `proxy-cpu-request` and `proxy-memory-request`) (thanks @TwinProduction!)
```

Comment:
Let's add `--` to the flags here to better indicate that they're flags. Something like:

```
  * Implemented `--proxy-cpu-limit` and `--proxy-memory-limit` flags for setting
    the proxy resources limits (`--proxy-cpu` and `--proxy-memory` were
    deprecated in favor of `--proxy-cpu-request` and `--proxy-memory-request`)
    (thanks @TwinProduction!)
```